[DEVCOMMS-483] Feature/localization section revamp (#1637)
mercadopago/devsite-docs: guides/resources/test-cards.en.md

# Local test cards

You can use test cards of local payment methods and simulate different payment responses, without the need to use a real card.

> WARNING
>
> Important
>

> >The DNI associated with the card must be the same as the test user making the purchase. To confirm this information, in your Mercado Pago user account, access **Your Profile > Your information** and review the **Document field**, ensuring that it will be the same to be associated with the card.

Reviewer: ```suggestion
>The DNI associated with the card must be the same as the test user making the purchase. To confirm this information, go to your Mercado Pago user account, access **Your Profile > Your information** and review the **Document field**, ensuring that it will be the same to be associated with the card.
```

> >The DNI associated with the card must be the same as the test user making the purchase. To confirm this information, go to your Mercado Pago user account, access **Your Profile > Your information** and review the **Document field**, ensuring that it will be the same to be associated with the card.

Author: ```suggestion
>The DNI associated with the card must be the same as the test user making the purchase. To confirm this information, go to your Mercado Pago user account, access **Your profile > Your information** and review the **Document** field, ensuring that it will be the same to be associated with the card.
```

> For this, depending on your country, use one of the cards that we provide below.

Reviewer: ```suggestion
For this, depending on your country, use one of the cards provided below.
```
